SonarQube.Common/Utilities.cs

```diff
@@ -198,6 +198,21 @@ public static void LogAssemblyVersion(ILogger logger, System.Reflection.Assembly
             logger.LogInfo("{0} {1}", description, assembly.GetName().Version);
         }
 
+        /// <summary>
+        /// Disposes the supplied object if it can be disposed. Null objects are ignored.
+        /// </summary>
+        public static void SafeDispose(object o)
+        {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) In method 'Utilities.SafeDispose(object)', consider providing a more meaningful name than parameter name 'o'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AIdentifiersShouldBeSpelledCorrectly)